File origsrc/moab-5.3.0/examples/makefile.config is a regular file while file src/moab-5.3.0/examples/makefile.config is a symbolic link
File origsrc/moab-5.3.0/itaps/iBase_f.h is a regular file while file src/moab-5.3.0/itaps/iBase_f.h is a symbolic link
--- origsrc/moab-5.3.0/itaps/imesh/Makefile.am	2021-07-14 00:57:13.000000000 +0900
+++ src/moab-5.3.0/itaps/imesh/Makefile.am	2021-09-09 06:56:34.262578400 +0900
@@ -78,6 +78,8 @@ libiMesh_la_LIBADD = $(top_builddir)/src
 
 libiMesh_la_LDFLAGS = -version-info @LIBTOOL_VERSION@
 
+libiMesh_la_LDFLAGS = -no-undefined
+
 # Automake doesn't seem to have a directory defined for
 # platform-dependent data (or include) files. So put
 # in $(libdir).  Define a $(cfgdir) to get around automake's
--- origsrc/moab-5.3.0/src/Makefile.am	2021-07-14 00:57:13.000000000 +0900
+++ src/moab-5.3.0/src/Makefile.am	2021-09-09 06:56:34.269588000 +0900
@@ -414,6 +414,8 @@ if HAVE_ZOLTAN
   libMOAB_la_LIBADD += $(ZOLTAN_LIBS) $(ZOLTAN_LIB_FLAGS)
 endif
 
+libMOAB_la_LDFLAGS = -no-undefined
+
 if HAVE_CGM
   AM_CPPFLAGS += @CGM_CPPFLAGS@
 endif
File origsrc/moab-5.3.0/src/moab/EntityHandle.hpp is a regular file while file src/moab-5.3.0/src/moab/EntityHandle.hpp is a symbolic link
File origsrc/moab-5.3.0/src/parallel/moab_mpi_config.h is a regular file while file src/moab-5.3.0/src/parallel/moab_mpi_config.h is a symbolic link
